infra/terraform-dev.tfvars

```diff
@@ -3,7 +3,7 @@ project      = "pagoda-discovery-platform-dev"
 docker_image = "us-east1-docker.pkg.dev/pagoda-discovery-platform-dev/mpc-recovery/mpc-recovery"
 
 account_creator_id           = "mpc-recovery-dev-creator.testnet"
-account_creator_sk_secret_id = "mpc-account-creator-sk-dev"
+account_creator_sk_secret_id = "mpc-recovery-account-creator-sk-dev"
```

Comment:
yeah, the issue was that it would break all the current PRs, that's why I went with a new name instead. But we can just explicitly say that we're using version 1 or something instead of latest that way this doesn't happen again

Reply:
Yeah that's what I assumed. We should make secrets parametrized by version here in the future for sure

Reply:
ok then, let's just explicitly put all versions then